core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.css

```diff
@@ -56,6 +56,7 @@ oppia-conversation-skin .supplemental-card-parent-container {
 }
 
 .conversation-skin-cards-container-embedded {
+  margin-bottom: 64px;
```

Comment:
To make this responsive, can we use percentages? Also, please add a video proof showing the UI is working for different screen sizes. Thank you!